infra/cifuzz/cifuzz_test.py

```diff
@@ -126,16 +126,48 @@ def test_valid(self):
               tmp_dir,
               commit_sha='0b95fe1039ed7c38fea1f97078316bfc1030c523'))
       self.assertTrue(os.path.exists(os.path.join(out_path, 'do_stuff_fuzzer')))
-      run_success, bug_found = cifuzz.run_fuzzers(5, tmp_dir)
-    self.assertTrue(run_success)
-    self.assertTrue(bug_found)
+
+      # Setting the first return to true than the second to false causes the
```

Comment:
nit: "Setting the first return value to True, then the second to False to emulate a bug existing in ...`

Reply:
Done.